src/styles/style-helpers.js

```diff
@@ -61,3 +61,35 @@ export const disableVisually = () => `
   opacity: 0.5;
   pointer-events: none;
 `;
+
+const add = (a, b) => a + b;
+const subtract = (a, b) => a - b;
+const multiply = (a, b) => a * b;
+const divide = (a, b) => a / b;
+
+const transformUnit = (values, transform, allowMultipleUnits = true) => {
+  const checkUnit = (aUnit, bUnit) => {
```

Comment:
Are you sure this should throw? Might completely break the style computation and leave users with no styles. Maybe instead this should return undefined, which only renders the one rule using this function invalid.

Reply:
Hm. Returning `undefined` or `''` is probably better. There still needs to be some kind of warning to the developer though. I'll think about this a bit more.